package.json

```diff
@@ -74,6 +75,7 @@
     "@wordpress/dependency-extraction-webpack-plugin": "^6.14.0",
     "@wordpress/dom-ready": "4.14.0",
     "@wordpress/e2e-test-utils-playwright": "^1.14.0",
+    "@wordpress/edit-post": "8.14.0",
```

Comment:
Seems like we aren't even using `@wordpress/edit-post` anymore? Remove.

Reply:
Good catch. Removed!